src/components/orgunits/OrgUnitDialog.js

```diff
@@ -141,7 +143,7 @@ export class OrgUnitDialog extends Component {
                             periodType={periodType}
                             period={period}
                             onChange={this.onPeriodChange}
-                            style={{ height: 70, margin: '-17px 0 0 3px' }}
+                            // style={{ height: 70, margin: '-12px 0 0 3px' }}
```

Comment:
Can we remove this instead of commenting or is it a placeholder for later?

Reply:
Ah, yes it should be removed. Fixing and merging.